Description: Don't set urgency hint if focused
 This patch, introduced in 9.05-2 didn't set the urgency hint if the window was
 focused. It was introducedas a "fix" to Debian bug 493981.
 .
 It was reverted by Ryan Kavanagh in 9.10-1 after discussion with upstream. The
 patch violates the ICCCM, which states:
 .
 ``The UrgencyHint flag, if set in the flags field, indicates that the client
 deems the window contents to be urgent, requiring the timely response of the
 user. [...] The window manager must also monitor the state of this flag for the
 entire time the window is in the Normal or Iconic state and must take
 appropriate action when the state of the flag changes. The flag is otherwise
 independent of the window's state.''
 .
 This means that if an application running in urxvt want the flag to be set, the
 flag should be set, regardless of the window's state / focus.
 .
 Further, it may be that the window is focused, but not visible, it which case
 the urgency flag is still useful.


 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
Author: Decklin Foster <decklin@red-bean.com>
Origin: vendor
Bug-Debian: http://bugs.debian.org/493981
Forwarded: no
Reviewed-By: Dropped by Ryan Kavanagh <ryanakca@kubuntu.org>
Last-Update: 2011-03-06
--- rxvt-unicode-9.09.orig/src/screen.C
+++ rxvt-unicode-9.09/src/screen.C
@@ -1928,7 +1928,7 @@ rxvt_term::scr_bell () NOTHROW
 # endif
 
 # if ENABLE_FRILLS
-  if (option (Opt_urgentOnBell))
+  if (option (Opt_urgentOnBell) && !focus)
     set_urgency (1);
 # endif
